### PR TITLE
[Merged by Bors] - chore: camel `cardinal_mk`

### PR DESCRIPTION
**Pull request by YaelDillies**
Rename `cardinal_mk` in lemma names to `cardinalMk` to follow the naming convention.

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/SetTheory/Cardinal/Basic.lean [hunk 1]
```diff
@@ -446,7 +446,7 @@ theorem le_one_iff_subsingleton {α : Type u} : #α ≤ 1 ↔ Subsingleton α :=
 theorem mk_le_one_iff_set_subsingleton {s : Set α} : #s ≤ 1 ↔ s.Subsingleton :=
   le_one_iff_subsingleton.trans s.subsingleton_coe
 
-alias ⟨_, _root_.Set.Subsingleton.cardinal_mk_le_one⟩ := mk_le_one_iff_set_subsingleton
+alias ⟨_, _root_.Set.Subsingleton.cardinalMk_le_one⟩ := mk_le_one_iff_set_subsingleton
```
Comment (b-mehta) [inline Mathlib/SetTheory/Cardinal/Basic.lean:449]: deprecated alias missing for this one

**Review by b-mehta — COMMENTED** [review]
bors d+

**Comment by vihdzp** [comment]
The fact that `#α` is precisely the constructor of `Cardinal` applied to `α` is not really intuitive unless you're aware of it. I'd also prefer that we simply rename these theorems to use `card`.

**Comment by YaelDillies** [comment]
> Did you see @eric-wieser 's suggestion on Zulip to rename it to something with `card` in name? I suggest that we make a decision first to avoid 2 renames in a row.

Yes I saw, but this is a much bigger rename than the one at hand, which is why I would rather do this one first. I have started [a Zulip thread](https://leanprover.zulipchat.com/#narrow/channel/287929-mathlib4/topic/Cardinal.2Emk) anyway.

**Comment by urkud** [comment]
@eric-wieser WDYT?

**Comment by urkud** [comment]
Eric says that he had no intention to block, so I'm re-delegating it. @YaelDillies , sorry for a delay caused by my misunderstanding.
bors d+

**Review by vihdzp — COMMENTED** [review]
Sorry to delay this even further, but I do have one ask.

File: Mathlib/SetTheory/Cardinal/Basic.lean (hunk 1 above)
Comment (YaelDillies) [reply Mathlib/SetTheory/Cardinal/Basic.lean:449]: cc @adomani, this one was missed by the script

**Comment by YaelDillies** [comment]
bors merge

**Comment by YaelDillies** [comment]
bors merge

File: Mathlib/SetTheory/Cardinal/Basic.lean (hunk 1 above)
Comment (adomani) [reply Mathlib/SetTheory/Cardinal/Basic.lean:449]: Ah, I had not thought of adding deprecations for *these* aliases!

**Pull request closed without merging**